### PR TITLE
Add fhir data quality deequ check

### PR DESCRIPTION
**Pull request by ibmatc**

### CODE REVIEW

File: services/fhir-data-quality/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+RUN mkdir /fhir-data-quality
+WORKDIR /fhir-data-quality
+ADD config config
+
+COPY config config
+COPY target/scala-2.12/fhir-data-quality-assembly*.jar fhir-data-quality.jar
+COPY fhir_data_quality.py .
+
+RUN wget -O spark.tgz https://dlcdn.apache.org/spark/spark-3.1.2/spark-3.1.2-bin-hadoop3.2.tgz;
```
Comment (ranum) [inline services/fhir-data-quality/Dockerfile:37]: I wonder if that semicolon is necessary.  I think it allows you to chain commands in a RUN but here there is only one? Not sure.

File: services/fhir-data-quality/src/main/resources/default.properties [hunk 2]
```diff
@@ -0,0 +1,16 @@
+## Data Quality configuration properties: ##
+
+# directory path of the FHIR bundles
+dataPath=/path/to/fhir-bundles
```
Comment (ranum) [inline services/fhir-data-quality/src/main/resources/default.properties:4]: I'm seeing some strange characters at the end of line 4,9, and 12.  Please check that there isn't anything there.  In the compare tool they show as ^M

**Review by ranum — COMMENTED** [review]
LGTM

**Review by ranum — APPROVED** [review]
LGTM

File: services/fhir-data-quality/src/main/resources/default.properties (hunk 2 above)
Comment (ibmatc) [reply services/fhir-data-quality/src/main/resources/default.properties:4]: I see the characters, but only in Git. I'm not sure exactly how to get rid of them, but I don't think they will cause any issue.  Going to leave them for now, as is.

**Pull request merged**